docker/secrets.env.sample

```diff
@@ -0,0 +1,4 @@
+# Use this file to store secrets used in Docker containers
```

Comment:
Maybe would be clear if we just define an `docker/.env` file to also store or override default vars.

The idea is to load it in our composition to patch "realtime" this values in all related services.

Reply:
I mean, I would also like to see here any composition vars, like `POSTGRES_HOST=db`, `POSTGRES_USER`, ...

What do you think @mcastellin?

Reply:
@XaviTorello you can now provide thos variables from the `docker/.env` file